javatar-security-spring-boot-starter/src/main/java/pro/javatar/security/starter/config/SecurityConfigImpl.java

```diff
@@ -0,0 +1,720 @@
+package pro.javatar.security.starter.config;
+
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.context.annotation.Configuration;
+import pro.javatar.security.api.config.SecurityConfig;
+
+import java.time.Duration;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+/**
+ * @author Borys Zora
+ * @version 2019-05-11
+ */
+@Configuration
+@ConfigurationProperties(prefix = "javatar.security")
+public class SecurityConfigImpl implements SecurityConfig {
```

Comment:
This class is too large.

Reply:
splitted, all subclasses moved to separate package config/model